Singer.API/Helpers/Exceptions/BadInputException.cs

```diff
@@ -2,49 +2,41 @@
 
 namespace Singer.Helpers.Exceptions
 {
-   /// <summary>
-   /// Exception that represents a 400 http status code.
-   /// </summary>
+   /// <summary>Exception that represents a 400 http status code.</summary>
```

Comment:
Faulty merge.

Reply:
how is this a faulty merge?

Reply:
If the message should change, what should it become?

Reply:
Message should be `Er is iets misgegaan.` (or alike)